### ISSUE TITLE
Cache cross toolchains on aarch64-darwin

### ISSUE THREAD

**Issue by szlend** [NixOS/nixpkgs#284465]
### Describe the bug

`aarch64-darwin` is slowly becoming the main darwin architecture. I think it would make sense to start binary caching cross-toolchains for it.

### Steps To Reproduce
1. `nix build -f . pkgsCross.aarch64-multiplatform.<pkg>`
2. It takes hours to bootstrap the build toolchain before building the actual package

### Expected behavior
The toolchain should be binary cached so it starts building `<pkg>` (or its dependencies) immediately.

### Additional context

We currently list `supportedSystems ? [ "x86_64-linux" "x86_64-darwin" "aarch64-linux" ]` in `release-cross.nix`. 

I presume we just need to add `aarch64-darwin` to it? I guess we also need to consider the additional build time this is going to take? I guess one alternative is also to drop `x86_64-darwin`.

### Notify maintainers

@reckenrode @wegank @Ericson2314 @sternenseemann @Artturin

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by siraben** [comment]
I agree with this, aarch64-darwin builders already are faster than x86_64-darwin on hydra.

**Comment by szlend** [comment]
Is it just me or is the `pkgsCross.aarch64-multiplatform` `stdenv` on the latest unstable cached now?
